lya/data.py

```diff
@@ -1,43 +1,73 @@
 from .conf import lya_data_path
-
+import numpy as np
+import matplotlib.pyplot as plt
 
 class Data(object):
-    def __init__(self, x, y, y_unc):
-        self.x = x
-        self.y = y
-        self.y_unc = y_unc
 
+    def __init__(self,
+                 k=None,
+                 P_k=None,
+                 P_k_unc=None,
+                 ):
 
-def load_lya_data(filename=lya_data_path):
-    """Load lyman-alpha data
+        self.k = k
+        self.P_k = P_k
+        self.P_k_unc = P_k_unc
```

Comment:
I thought about this, but let's stick with the generic names `x` and `y`.  I like the reminder that there's nothing special to the particular problem about what we're doing (could just as easily be any other kind of data or any other kind of model)